documentation/release_notes.rst

```diff
@@ -60,7 +60,7 @@ See oneDPL Guide for other `restrictions and known limitations`_.
 - With libstdc++ version 10, the compilation error *SYCL kernel cannot use exceptions* occurs
   when calling the range-based ``adjacent_find``, ``is_sorted`` or ``is_sorted_until`` algorithms with device policies.
 - The range-based ``count_if`` may produce incorrect results on Intel® Data Center GPU Max Series when the driver version
-  is "Rolling 2507.12" and newer.
+  is *Rolling 2507.12* or *Rolling 2507.17*. Other *Rolling* and all *LTS* versions are unaffected.
```

Comment:
Both these rolling releases are not supported by now, see https://dgpu-docs.intel.com/releases/releases.html. Does it still make sense to keep the note?

Reply:
Well, removing an issue that became practically irrelevant is also an update :)

Eventually we will need to remove the note. What would be a trigger for that, general re-evaluation of known issues before the next release?

Reply:
Let's remove it here.

I was concerned about what a user sees in https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-dpcpp-library-release-notes.html. If note will be updated there for 2022.9, there is nothing to do after deleting it here. If it will not, I still think that we should say something that now count_if works with the relevant drivers for 2022.10 as an update.

Reply:
Thanks.
I will keep this conversation open (unresolved) but only for the visibility by other reviewers.